react-components/src/components/Image360CollectionContainer/Image360CollectionContainer.tsx

```diff
@@ -115,6 +116,18 @@ export function Image360CollectionContainer({
   }
 }
 
+const useSetIconCulling = (
+  collection?: Image360Collection,
+  cullingParameters?: { radius?: number; iconCountLimit?: number }
+): void => {
+  useEffect(() => {
+    collection?.set360IconCullingRestrictions(
+      cullingParameters?.radius ?? Infinity,
+      cullingParameters?.iconCountLimit ?? 50
+    );
+  }, [collection, cullingParameters?.radius, cullingParameters?.iconCountLimit]);
```

Comment:
`useEffect` will get rigger every time `cullingParameters` object changes even if `cullingParameters.radius` & `cullingParameters.iconCountLimit` is not changed